pallets/ocex/src/tests.rs

```diff
@@ -2445,28 +2455,29 @@ pub fn test_set_balances_when_bounded_vec_limits_out_of_bound() {
 	});
 }
 
-#[test]
-pub fn test_set_balances_when_bounded_vec_limits_in_bound() {
-	let account_id = create_account_id();
-	new_test_ext().execute_with(|| {
-		assert_ok!(OCEX::set_exchange_state(RuntimeOrigin::root(), false));
-		let mut vec_of_balances: Vec<HandleBalance<AccountId32>> = vec![];
-		for _i in 0..1000 {
-			vec_of_balances.push(HandleBalance {
-				main_account: account_id.clone(),
-				asset_id: AssetId::Polkadex,
-				free: 100,
-				reserve: 50,
-			});
-		}
-		let bounded_vec_for_alice: BoundedVec<HandleBalance<AccountId>, HandleBalanceLimit> =
-			BoundedVec::try_from(vec_of_balances).unwrap();
-		assert_eq!(
-			OCEX::set_balances(RuntimeOrigin::root(), bounded_vec_for_alice.clone()),
-			Ok(())
-		);
-	});
-}
+//FIXME: This test case is failing. Check if it relevant ot not
```

Comment:
Please do. But I assume it is not, since extrinsic was removed by @Gauthamastro [here](https://github.com/Polkadex-Substrate/Polkadex/pull/884/files#diff-2ff81ab6e292e9586175a812766f54c0f78ce2d05d3eddc851c4da48b6cf3a15L866-L892).